C/main_libFuzzer_extra_counters.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <string.h>
+
+typedef long long GoInt64;
+typedef GoInt64 GoInt;
+typedef struct { void *data; GoInt len; GoInt cap; } GoSlice;
+extern void gofuzz_Init(void* coverTabPtr, uint64_t coverTabSize);
+extern void gofuzz_Run(GoSlice p0);
+
+
+#ifdef __linux__
```

Comment:
This will silently break on other OSes with obscure failure mode. It's better to make the build fail loudly instead.

Reply:
Addressed in latest commit.

Reply:
Push? I don't see it yet.

Reply:
If we are looking at the same source (which is not always clear on github), then there is below:
```
#else
#error Currently only Linux is supported
```